grafana/resource_alert_notification.go

```diff
@@ -38,6 +38,12 @@ func ResourceAlertNotification() *schema.Resource {
 				Optional:  true,
 				Sensitive: true,
 			},
+
+			"uid": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Required: false,
```

Comment:
You don't need Required next to Optional here. Replace it with `Computed: true`.

Reply:
Thanks! Changes made.